### PR TITLE
Compare more models across frame and tabular

### PR DESCRIPTION
**Pull request by wsad1**
Follow up to #398. 
Adds an option to compare `FTTransformer` accross pytorch tabular and frame.

CUDA: 
   Frame: 43.7 it/s
   Tabular: 40.15 it/s

### CODE REVIEW

File: benchmark/pytorch_tabular_benchmark.py [hunk 1]
```diff
@@ -30,7 +30,7 @@
 parser.add_argument('--batch_size', type=int, default=256)
 parser.add_argument('--epochs', type=int, default=1)
 parser.add_argument('--model_type', type=str, default='TabTransformer',
-                    choices=['TabTransformer'])
+                    choices=['TabTransformer', 'FTTransformer'])
```
Comment (yiweny) [inline benchmark/pytorch_tabular_benchmark.py:33]: Is it possible to include TabNet?

File: benchmark/pytorch_tabular_benchmark.py [hunk 2]
```diff
@@ -108,6 +129,7 @@ def train_tabular_model() -> float:
     )
     end = time.time()
     tabular_train_time = end - start
+    print(tabular_model.model)
```
Comment (yiweny) [inline benchmark/pytorch_tabular_benchmark.py:132]: Remove or make it more meaningful?

File: benchmark/pytorch_tabular_benchmark.py [hunk 3]
```diff
@@ -179,11 +210,13 @@ def test(loader: DataLoader) -> float:
 
     end = time.time()
     frame_train_time = end - start
+    print(model)
```
Comment (yiweny) [inline benchmark/pytorch_tabular_benchmark.py:213]: Same

**Review by yiweny — APPROVED** [review]
Thanks a lot!

File: benchmark/pytorch_tabular_benchmark.py (hunk 2 above)
Comment (wsad1) [inline benchmark/pytorch_tabular_benchmark.py:132]:
```suggestion
```

File: benchmark/pytorch_tabular_benchmark.py (hunk 3 above)
Comment (wsad1) [inline benchmark/pytorch_tabular_benchmark.py:213]:
```suggestion
```

File: benchmark/pytorch_tabular_benchmark.py (hunk 1 above)
Comment (wsad1) [reply benchmark/pytorch_tabular_benchmark.py:33]: The tab net implementation in tabular is super different between pyg and frame. I can't get the models to have approximate the same number of parameters.

**Review by akihironitta — APPROVED** [review]
LGTM!

File: benchmark/pytorch_tabular_benchmark.py [hunk 4]
```diff
@@ -9,15 +9,14 @@
 import torch.nn.functional as F
 from pytorch_tabular import TabularModel
 from pytorch_tabular.config import DataConfig, OptimizerConfig, TrainerConfig
-from pytorch_tabular.models.common.heads import LinearHeadConfig
-from pytorch_tabular.models.tab_transformer import TabTransformerConfig
+from pytorch_tabular.models import FTTransformerConfig, TabTransformerConfig, LinearHeadConfig
```
Comment (wsad1) [inline benchmark/pytorch_tabular_benchmark.py:12]:
```suggestion
from pytorch_tabular.models import (FTTransformerConfig, TabTransformerConfig, LinearHeadConfig,)
```

**Pull request merged**